spectrometer.cabal

```diff
@@ -178,6 +178,12 @@ library
     App.Fossa.VSI.IAT.Types
     App.Fossa.VSI.Types
     App.Fossa.VSIDeps
+    App.NewFossa.Config.Analyze
+    App.NewFossa.Config.Common
+    App.NewFossa.ConfigFile
```

Comment:
[optional] [nit] prefer `ConfigFile` and `EnvironmentVars ` modules in config namespace

Reply:
Moved to #756